Makefile

```diff
@@ -19,6 +19,7 @@ endif
 
 build:
 	$(docker-compose) build
+	docker-compose run web bash -c "bundle exec rails db:migrate && bundle exec rails db:seed"
```

Comment:
This should probably use `$(docker-compose)` and not just `docker-compose`.  Running seed from the build command though can lead down a tricky slope with duplicate entries if we're not careful.